mm/2s2h/Rando/GiveItem.cpp

```diff
@@ -98,6 +98,14 @@ void Rando::GiveItem(RandoItemId randoItemId) {
         case RI_PROGRESSIVE_BOW:
             Rando::GiveItem(Rando::ConvertItem(randoItemId));
             break;
+        case RI_GS_TOKEN_SWAMP:
+            SET_QUEST_ITEM(randoItemId - ITEM_SKULL_TOKEN + QUEST_QUIVER);
```

Comment:
I'm not sure I understand this, can you just do `SET_QUEST_ITEM(QUEST_SKULL_TOKEN)`?

Reply:
That came from z_parameter.c, which had this comment: 

`//! @bug: Sets QUEST_QUIVER instead of QUEST_SKULL_TOKEN`
`// Setting 'QUEST_SKULL_TOKEN' will result in misplaced digits on the pause menu - Quest Status page.`

I thought it would best to keep it consistent, but I can change it if you'd like.

Reply:
Lets just do `SET_QUEST_ITEM(QUEST_QUIVER);` 👍